### ISSUE TITLE
Array.from causes issues

### ISSUE THREAD

**Issue by dxxvi** [DevExpress/testcafe#4137]
```javascript
import { ClientFunction } from 'testcafe';

fixture `Getting started`.page `https://www.google.com`

const clientFunction = ClientFunction(() => {
  return Array.from(document.querySelectorAll('div')).length;
});

test('test', async t => {
  const x = await clientFunction();
  console.log(x);
});
```

**Comment by helen-dikareva** [comment]
Thank you for reaching us and posting this information. 
It seems we have some problem with the compilation of ClientFunction's code with the babel. We'll investigate and fix it.
 
I know your example is just for demonstration purposes, but I want to mention that you can use `document.querySelectorAll('div').length` in the `ClientFunction`. And the best way to get the number of elements on the page is to use [Selector's `count`](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/using-selectors.html#check-if-an-element-exists) property: `Selector('div').count`.

**Comment by joywave** [comment]
Also seeing this issue, my workaround was to use `Array.prototype.slice.call()`

**Comment by devmondo** [comment]
@joywave `querySlectorAll` returns a node list and it is not really a typical array. 
you can convert it to array using 

```js
const divs= [...document.querySelectorAll('div')];
console.log(divs.length);
```

**Comment by miherlosev** [comment]
The problem is fixed in the `testcafe@1.10.1` version.
